build.sbt

```diff
@@ -638,6 +687,20 @@ lazy val hasLargeIntegrationTests = Seq(
   javaOptions in (Test, run) += "-Xss4m"
 )
 
+lazy val nativeSettings = Seq(
+  scalaVersion := LatestScala211,
+  crossScalaVersions := List(LatestScala211),
+  nativeGC := "immix",
+  nativeMode := "release",
+  // TODO: I'd love to switch nativeLinkStubs to false, but I can't.
+  // No idea where the following nativeLink error is coming from.
+  // [error] cannot link: @java.lang.Thread::getStackTrace_scala.scalanative.runtime.ObjectArray
+  // [error] unable to link
+  // [error] (testsNative/nativetest:nativeLinkNIR) unable to link
+  // [error] Total time: 44 s, completed Feb 1, 2018 3:59:15 PM
+  nativeLinkStubs := false
```

Comment:
@densh Are there best practices for debugging linkage errors? I can't seem to find out who uses Thread.getStackTrace. I'm reasonably sure that the direct caller is not in Scala Native core libs or in Scalameta, but I've no idea how to pinpoint the problematic 3rd-party library without bruteforcing through everything.

Reply:
This is probably ScalaTest. When porting over to Scala Native I was seeing these linking errors and enabled `nativeLinkStubs` to get around them, but never got to actually finding where this method was used. I'll try to do some more investigation to find the source.

Reply:
@shadaj Thank you for your work! I really appreciate that we now have a native-compatible build of ScalaTest 💯

Reply:
@xeno-by just pushed https://github.com/scalatest/scalatest/pull/1322, which should make it possible to run tests without linking stubs!